node/readme.md

```diff
@@ -17,29 +17,31 @@ import * as broysterForNode from '@fpjs-incubator/broyster/node'
 
 This package exports the following:
 
-- `@fpjs-incubator/broyster/node`:
-    - `karmaPlugin` That can be used for launching and reporting tests.
-    - `sslConfiguration` That provides a self-signed certificate for HTTPS testing on localhost.
-    - `httpHttpsServer` That gives you a set of two servers - one with HTTP and one with HTTP capabilities.
-        Newer versions of Safari do not work nor have workarounds for self-signed certificates, however their behavior is the same for both HTTP and HTTPS. Depending on your entry's *useHttps*, the launcher will redirect respectively.
+-   `@fpjs-incubator/broyster/node`:
+    -   `karmaPlugin` That can be used for launching and reporting tests.
+    -   `sslConfiguration` That provides a self-signed certificate for HTTPS testing on localhost.
+    -   `httpHttpsServer` That gives you a set of two servers - one with HTTP and one with HTTP capabilities.
+        Newer versions of Safari do not work nor have workarounds for self-signed certificates, however their behavior is the same for both HTTP and HTTPS. Depending on your entry's _useHttps_, the launcher will redirect respectively.
         The HTTP server runs on the port provided by Karma, while the HTTPS port will run on +1 from that.
-    - `setHttpsAndServerForKarma` That configures karma for HTTP and HTTPS testing without any additional work.
-- `@fpjs-incubator/broyster/browser`:
-    - `retryFailedTests` That allows overriding the different behavior of Jasmine specs. The new behavior will retry a failed test up until the maximum specified in the first parameter, with a delay between each such attempt, indicated by the second parameter (in miliseconds). Call this function in the root of any executable file, involved in your testing code, for example, in a Jasmine helper file. Once called, it affects all tests Jasmine runs, even in the other files. For Karma, you can add a file that contains the invocation and point it in your `files`, that way you will not have it tied to one specific test file.
+    -   `setHttpsAndServerForKarma` That configures karma for HTTP and HTTPS testing without any additional work.
+    -   `CapabilitiesFactory` Stores credentials for BrowserStack
```

Comment:
The caps factory actually creates an object that describes the parameters of the driver session that will be requested